src/xdplwf/oid.c

```diff
@@ -266,7 +354,7 @@ XdpLwfCommonOidRequestInspectComplete(
 
     if (Status != NDIS_STATUS_PENDING) {
         if (ClonedRequest != NULL) {
-            XdpLwfOidRequestComplete(Filter, ClonedRequest, Status);
+            XdpLwfCommonOidRequestComplete(Filter, RequestInterface, ClonedRequest, Status);
```

Comment:
This is the only functional change. The rest is logging or necessary reordering of functions.